app/assets/javascripts/settings-controller.coffee

```diff
@@ -60,35 +62,57 @@ template = """
       <option value={{code}}>{{description}}</option>
       {{/}}
     </select>
-  </div>                           
+  </div>
 </div>
 
 
 <h1 class="settings-subheader">Works in Progress</h1>
-<a><center> Here you can access all the models that have been edited</center></a>
-  
+<div class="description">Here are all the models you have made changes to in NetLogo Web.</div>
+
 <ul>
   {{#each workInProgressLinks}}
-    <li> <a href="{{url}}">{{modelTitle}}</a> <br>Date Last Modified: {{dataAccessed}}</li>
-    <br>
+    <li><a href="{{url}}">{{modelTitle}}</a> {{storageTag}} <div>{{dateAccessed}} at {{timeAccessed}}</div></li>
   {{/each}}
 </ul>
 
 """
+#Formatting Links for HTML display
 formatLinks = (wipStorage) ->
   result = []
```

Comment:
Nit: Since this variable holds multiple values for display, maybe pluralizing it `results` would be clearer?